### PR TITLE
[BEAM-2853] Move the nexmark documentation to the website

### PR DESCRIPTION
**Pull request by echauchot**
I presented it as a benchmark solution that can target all the Beam supported execution engines. Please give me your opinion if this public communication is suited.

R: @jbonofre 
R: @reuvenlax

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/708/

Jenkins built the site at commit id 819be0425df9b7231272c6f962e4f3dff599ead4 with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/323/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by echauchot** [comment]
@jbonofre, @reuvenlax do you have any comments on this online doc?

**Comment by aaltay** [comment]
@echauchot Is this a plain copy of the [readme](https://github.com/apache/beam/blob/dee8e93e771c90fe6d9207fe65b01db8c7ea41a9/sdks/java/nexmark/README.md) file? 

cc: @melap

**Comment by echauchot** [comment]
Hi @aaltay no it contains almost all the README but adds things about workload configuration, output of nexmark, and a bit of reoganisation of the outline to make it more readable

**Comment by aaltay** [comment]
The staged version expired, I could not verify how it looks. Content looks good to me. Let's give @melap a chance to add comments and we can merge after that.

**Comment by melap** [comment]
retest this please

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/733/

Jenkins built the site at commit id 819be0425df9b7231272c6f962e4f3dff599ead4 with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/323/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

File: src/documentation/sdks/nexmark.md [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+## Benchmark launch configuration
+
+We can specify the Beam runner to use with maven profiles, available profiles are:
+
+    direct-runner
+    spark-runner
+    flink-runner
+    apex-runner
+
+The runner must also be specified like in any other Beam pipeline using
```
Comment (melap) [inline src/documentation/sdks/nexmark.md]: (throughout this page) if there's a partial sentence before a command line/code block, would be good to add : at the end  e.g. using:

**Comment by echauchot** [comment]
Thanks everyone for the review! I addressed the comments.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/734/

Jenkins built the site at commit id 59eefacb04a5fd245e55d6dbecbedf5f0630ee46 with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/323/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Pull request closed without merging**